src/visual.ts

```diff
@@ -412,8 +413,30 @@ module powerbi.extensibility.visual {
                 this.data.dateValues,
                 (dateValue: DateValue) => dateValue.value);
 
-            const minDate: number = extentDate[0];
-            const maxDate: number = extentDate[1] + (extentDate[1] - extentDate[0]) * LineDotChart.dateMaxCutter;
+            const columnFormattingFn = (index: number, dataType: valueType): any => {
```

Comment:
Let's not to create anonymous function. It'd be better to convert it to a method and use bind in constructor like we do in React